presentation/backend/src/presentation-backend/PresentationManagerDetail.ts

```diff
@@ -73,6 +74,7 @@ export class PresentationManagerDetail implements IDisposable {
 
     this._onManagerUsed = undefined;
     this.rulesets = new RulesetManagerImpl(getNativePlatform);
+    this._diagnosticsCallback = params.diagnosticsCallback;
```

Comment:
It's not clear to me how the consumer would associate the `ReadableSpan[]` he gets through this callback with the active RPC request. Do we have a plan for that?

Reply:
Please add a section to `NextVersion.md` with an example for how to set OpenTelemetry reporting.